db/db_impl/db_impl_write.cc

```diff
@@ -1651,12 +1651,7 @@ Status DBImpl::TrimMemtableHistory(WriteContext* context) {
   for (auto& cfd : cfds) {
     autovector<MemTable*> to_delete;
     bool trimmed = cfd->imm()->TrimHistory(
-        &to_delete, cfd->mem()->ApproximateMemoryUsage());
-    if (!to_delete.empty()) {
```

Comment:
It sort of confuses me what would happen if another thread installs a new SuperVersion while a TrimHistory is in progress (after MemTableList updated to a new MemTableListVersion but before it mutates memlist_history_. It's not really related to this PR though.

Reply:
Oh this function is under DB mutex. That's good